packages/components/src/controllers/useSizeOverride.ts

```diff
@@ -0,0 +1,94 @@
+import { makeController } from "@arcgis/lumina/controllers";
+
+/**
+ * Axis used when applying a size override.
+ * "inline" maps to width, "block" maps to height.
```

Comment:
> "inline" maps to width, "block" maps to height.

Let's omit this. Technically, the context of `inline` and `block`-related CSS props can change w/ `writing-mode`.